src/lint/lint.ts

```diff
@@ -322,22 +339,49 @@ async function fix(
   const {appliedEdits, incompatibleEdits, editedFiles} =
       await applyEdits(edits, makeParseLoader(analyzer, analysis));
 
-  for (const [newPath, newContents] of editedFiles) {
-    await fs.writeFile(
-        path.join(config.root, newPath), newContents, {encoding: 'utf8'});
+  const pathToFileMap = new Map<string, string>();
+  for (const [url, newContents] of editedFiles) {
+    const conversionResult = urlLoader.getFilePath(url);
+    if (conversionResult.successful === false) {
+      console.log(
```

Comment:
should we use `logger` here and below?

Reply:
+1 to this fatal error. The rest of the output shouldn't go through the logger I don't think, it should be logged directly to stdout as it's the primary purpose of running the command to log this stuff to stdout.